contracts/staking/RewardsDistributorAdmin.sol

```diff
@@ -0,0 +1,83 @@
+pragma solidity ^0.8.0;
+
+import "./TribalChief.sol";
+import "../refs/CoreRef.sol";
+import "../external/rari/IRewardsDistributor.sol";
+
+contract RewardsDistributorAdmin is IRewardsDistributor, CoreRef {
+    /// rewards distributor contract address
+    IRewardsDistributor public rewardsDistributorContract;
+
+    constructor(
+        address coreAddress,
+        IRewardsDistributor _rewardsDistributorContract
+    ) CoreRef(coreAddress) {
+        rewardsDistributorContract = _rewardsDistributorContract;
+        // Create new admin role for this contract
+        // grant this role to all AutoRewardsDistributors so that they can call 
+        _setContractAdminRole(keccak256("REWARDS_DISTRIBUTOR_ADMIN_ROLE"));
+    }
+
+    /**
+      * @notice Begins transfer of admin rights. The newPendingAdmin must call `_acceptAdmin` to finalize the transfer.
+      * @dev Admin function to begin change of admin. The newPendingAdmin must call `_acceptAdmin` to finalize the transfer.
+      * @param newPendingAdmin New pending admin.
+      */
+    function _setPendingAdmin(address newPendingAdmin) external override onlyGovernorOrAdmin {
+        rewardsDistributorContract._setPendingAdmin(newPendingAdmin);
+    }
+
+    /**
+      * @notice Accepts transfer of admin rights. msg.sender must be pendingAdmin
+      * @dev Admin function for pending admin to accept role and update admin
+      */
+    function _acceptAdmin() external override onlyGovernorOrAdmin {
+        rewardsDistributorContract._acceptAdmin();
+    }
+
+    /*** Comp Distribution ***/
+   /*** Comp Distribution Admin ***/
+
+    /**
+     * @notice Transfer COMP to the recipient
+     * @dev Note: If there is not enough COMP, we do not perform the transfer all.
+     * @param recipient The address of the recipient to transfer COMP to
+     * @param amount The amount of COMP to (possibly) transfer
+     */
+    function _grantComp(address recipient, uint amount) external override onlyGovernorOrAdmin {
```

Comment:
this should be exclusively `onlyGovernor` admin shouldn't be able to withdraw TRIBE